src/tuning/kernels/xgemm.cpp

```diff
@@ -63,7 +63,7 @@ class TuneXgemm {
   static double DefaultInfluenceGlobalPSO(){ return 0.1; }
   static double DefaultInfluenceLocalPSO(){ return 0.3; }
   static double DefaultInfluenceRandomPSO(){ return 0.6; }
-  static size_t DefaultHeuristic(){ return size_t{3};} // PSO
+  static size_t DefaultHeuristic(){ return (size_t)cltune::SearchMethod::PSO;} // PSO
```

Comment:
Yes indeed, you are right. Perhaps I'll work on a tuner-specific utilities file and then we can do it later :-)

Small thing: could you use new C++11 style casts? So use `static_cast<size_t>(var_name)` instead? Thanks!